web-back/lore/index.ts

```diff
@@ -8,7 +8,7 @@ import { loreRepo } from "db";
 app.get("/api/lore", async () => {
     const results = await loreRepo.list();
     if (!results) {
-        return jsonResponse(404, { error: "No results" });
+        return jsonResponse(404, { error: "No results" }); // Not sure that not having any lore yet is an error. 204 is probably better.
```

Comment:
I guess it is still an error. According to the docs, 204 means success without body : https://developer.mozilla.org/fr/docs/Web/HTTP/Status/204

Reply:
Maybe 200 with a `[]` is better then.

Reply:
Agree :+1: